spec/factories/temporary_files.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :temporary_file do
+    
```

Comment:
@jtu0 Super nit: might be more reflective of real-world data to set contents to empty string by default instead of letting them be nil. 

The biggest impact would be if we ever call string methods on the `contents` for this factory without re-specifying the data - an unreplaced nil would NoMethod but empty string wouldn't - not sure which is preferred.

Reply:
On second thought though, IF we want behavior like the latter, I think we should consider throwing a factory-level error saying to specify it

Reply:
I like this idea - going to merge this now so @corneliusellen will be able to demo with the stakeholder this week, if I had time I would do this though!